tests/integration_tests/core/conftest.py

```diff
@@ -0,0 +1,90 @@
+import os
+from typing import Iterator, Text, Type
+
+import pytest
+from _pytest.tmpdir import TempdirFactory
+import sqlalchemy as sa
+
+from rasa.core.agent import Agent
+from rasa.core.lock_store import LockStore, RedisLockStore
+from rasa.core.policies.memoization import AugmentedMemoizationPolicy
+from rasa.core.policies.rule_policy import RulePolicy
+from rasa.core.tracker_store import InMemoryTrackerStore
+from rasa.shared.core.domain import SessionConfig
+
+
+REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
+REDIS_PORT = os.getenv("REDIS_PORT", 6379)
+POSTGRES_HOST = os.getenv("POSTGRES_HOST", "localhost")
+POSTGRES_PORT = os.getenv("POSTGRES_PORT", 5432)
+POSTGRES_USER = os.getenv("POSTGRES_USER", "")
+POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD", "")
+POSTGRES_DEFAULT_DB = os.getenv("POSTGRES_DEFAULT_DB", "postgres")
+POSTGRES_TRACKER_STORE_DB = "tracker_store_db"
+POSTGRES_LOGIN_DB = "login_db"
+
+DEFAULT_STORIES_FILE = "data/test_yaml_stories/stories_defaultdomain.yml"
+
+
+@pytest.fixture
+def redis_lock_store() -> RedisLockStore:
+    return RedisLockStore(REDIS_HOST, REDIS_PORT)
```

Comment:
Is it worth making a redis fixture that cleans up data?

Reply:
yes! and it's dead simple with Redis